src/translators/errors.py

```diff
@@ -51,18 +51,23 @@ def __init__(self, error: Exception):
     def error(self) -> Exception:
         return self._error
 
-    def is_transient_error(self) -> bool:
+    def is_transient_error(self):
```

Comment:
Rather than changing the logic of transient error function, I would add a new function called ‘is_aggregation_error’

Reply:
done

Reply:
`def is_transient_error(self) -> bool:`